### ISSUE TITLE
Error while running "yum install kaltura-server" on RHEL X86_64

### ISSUE THREAD

**Issue by kcsunkara** [kaltura/platform-install-packages#159]
Hello,

I'm trying to install Kaltura server on RHEL.
MySQL Details:

> mysql --version
> mysql  Ver 14.14 Distrib 5.5.37, for Linux (x86_64) using readline 5.1

While running the below script and after answering the questions, I'm getting the following error:

> /opt/kaltura/bin/kaltura-config-all.sh 

Starting monit: Starting monit daemon with http interface at [*:2812]
 [  OK  ]
Configuring your Kaltura DB...

Checking MySQL version..
Your version of MySQL is not compatible with Kaltura at the moment. 
Please install and configure MySQL 5.1 according to the instructions on the Kaltura install manual before proceeding with the Kaltura installation.
ERROR: we failed on something else..

Do I have to uninstall 5.5 ? or is there any workaround.

Regards,
Krishna Sunkara.

**Comment by cvideos** [comment]
Hi m trying to install kaltura 10 in fedora 21 without ssl
first i got error with mysql version..
then I installed mysql5.1 using 
yum install MySQL*RPM
It is installed..but when i run 
kaltura-config-all.sh
im getting errors like this

Redirecting to /bin/systemctl restart  httpd.service
Note: Forwarding request to 'systemctl enable httpd.service'.
Restarting kaltura-monit (via systemctl):                  [  OK  ]
Running Sphinx config...


kaltura-sphinx-2.2.1-16.x86_64
base-config completed successfully, if you ever want to re-configure your system (e.g. change DB hostname) run the following script:
# rm /opt/kaltura/app/base-config.lock
# /opt/kaltura/bin/kaltura-base-config.sh


Starting kaltura-monit (via systemctl):                    [  OK  ]


Configuring your Kaltura DB...


Checking MySQL version..
mysql: unknown option '--innodb_file_per_table'



		Archving logs to /opt/kaltura/log/log_29_04_15_10_20.tar.gz...
ERROR: we failed on something else..
I changed  in kaltura-mysql-settings.sh as service mysql restart.. previously it was service mysqld restart
thanks
could u help me

**Comment by cvideos** [comment]
I also tried removing base config lock and running it again 
i also tried dropping data bases..but it s not working..

**Comment by jessp01** [comment]
Hello,

Can you provide the exact name and version of the MySQL RPM you used? we 
have tested using the mysql-server package in the official base repo of 
CentOS 6 and 7, which is the same as in RHEL. We do not officially test on 
FC although there is no reason why it shouldn't work.

May the source be with you,

Jess Portnoy

Join us for the Kaltura Connect Virtual Summits 2015 - Register now!
connect.kaltura.com

**Comment by cvideos** [comment]
Hello 
Thanks for ur response. 
I 've downloaded mysql 5. 1. 73. rhel. Rpm as a bundle  from mySql official page and I installed using rpm command in command line.. but now my admin console s working.. I think it s not a complete installation.. coz when I log in to my Kmc page I'm getting uiconf o error.. I'm expecting a good solution from u. During installation after creating database and connectivity testpassed it's not going.. I'm getting error like we failed on some thing else.. but I'm getting admin console.. problem with kmc now.. 
Thanks 
Sent from my Samsung Galaxy smartphone.

-------- Original message --------
